app/build.gradle

```diff
@@ -109,7 +109,7 @@ android {
         targetSdkVersion 28
         multiDexEnabled true
         versionCode 1500
-        version "2.6-dev"
+        version "2.6-dev-omnicore"
```

Comment:
This is not OmniCore, but the native Java driver. I think 2.6-dev-omnipod would be a more appropriate name